.changeset/green-cheetahs-cover.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-app-api': minor
```

Comment:
Actually!  In addition to this changeset, you might add the notes to the `navigate` row in the `Key Events` table in the documentation: https://backstage.io/docs/plugins/analytics#key-events